include/io/Display.hpp

```diff
@@ -17,10 +18,14 @@ namespace chip8::io
 
             class PixelColumn 
             {
-                friend DisplayPixels;
                 public:
+                    friend DisplayPixels;
```

Comment:
I think the need to. PixelColumn needs to access the pixels array, which is owned by the matrix. This sub class is a 'proxy class'. I based this on this answer:

https://stackoverflow.com/questions/15762878/c-overload-operator

But this might change if we change the interface to something more 'iterator-compliant'